shared/versions/stack/7.8.asciidoc

```diff
@@ -0,0 +1,31 @@
+:version:                7.8.0
+////
+bare_version never includes -alpha or -beta
+////
+:bare_version:           7.8.0
+:logstash_version:       7.8.0
+:elasticsearch_version:  7.8.0
+:kibana_version:         7.8.0
+:apm_server_version:     7.8.0
+:branch:                 7.x
```

Comment:
`branch` is used to populate links in the [shared attributes](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc) file. These APM links appear to be failing because they are linking to `7.x` which doesn't exist. All of the other documentation works fine because they build a `7.x` branch. I think this needs to be updated, not just to fix the APM links, but to make sure all links in 7.8 point to `7.8`.

```suggestion
:branch:                 7.8
```